packages/in_app_purchase/tizen/src/billing_manager.h

```diff
@@ -0,0 +1,183 @@
+// Copyright 2023 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_PLUGIN_BILLING_MANAGER_H
+#define FLUTTER_PLUGIN_BILLING_MANAGER_H
+
+#include <flutter/encodable_value.h>
+#include <flutter/method_channel.h>
+#include <flutter/plugin_registrar.h>
+#include <tizen_error.h>
+
+#include <iostream>
+#include <map>
+#include <string>
+#include <vector>
+
+#include "billing_service_proxy.h"
+
+#define SSO_API_MAX_STRING_LEN 128
+
+typedef struct sso_login_info {
+  char login_id[SSO_API_MAX_STRING_LEN];
+  char login_pwd[SSO_API_MAX_STRING_LEN];
+  char login_guid[SSO_API_MAX_STRING_LEN];
+  char uid[SSO_API_MAX_STRING_LEN];
+  char user_icon[SSO_API_MAX_STRING_LEN * 8];
+} sso_login_info_s;
+
+typedef enum {
+  PRD = 0,
+  DEV,
+} tv_server_type;
+
+typedef enum {
+  SYSTEM_INFO_ERROR_NONE = TIZEN_ERROR_NONE, /**< Successful */
+  SYSTEM_INFO_ERROR_INVALID_PARAMETER =
+      TIZEN_ERROR_INVALID_PARAMETER, /**< Invalid parameter */
+  SYSTEM_INFO_ERROR_OUT_OF_MEMORY =
+      TIZEN_ERROR_OUT_OF_MEMORY, /**< Out of memory */
+  SYSTEM_INFO_ERROR_IO_ERROR =
+      TIZEN_ERROR_IO_ERROR, /**< An input/output error occurred when reading
+                               value from system */
+  SYSTEM_INFO_ERROR_PERMISSION_DENIED =
+      TIZEN_ERROR_PERMISSION_DENIED, /**< No permission to use the API */
+  SYSTEM_INFO_ERROR_NOT_SUPPORTED =
+      TIZEN_ERROR_NOT_SUPPORTED, /**< Not supported parameter (Since 3.0) */
+} system_info_error_e;
```

Comment:
'system_info_error_e' is inclued in head file 'system_info.h', why not use system_info.h'?